kv/keyflags.go

```diff
@@ -45,10 +45,27 @@ const (
 	flagNeedCheckExists
 	flagPrewriteOnly
 	flagIgnoredIn2PC
+	flagAssertExist
+	flagAssertNotExist
```

Comment:
🤔 How about making a 2-bit flag rather than two 1-bit flags so that it can represent up to 4 different choices